pkg/controller/bootimage/boot_image_controller.go

```
@@ -111,10 +133,21 @@ func (mrs MachineResourceStats) getDegradedStatusMessage(name string) string {
 }
 
+// GVRs for CAPI core resources and per-platform infrastructure templates
+var (
+	capiMachineSetGVR        = clusterv1.GroupVersion.WithResource("machinesets")
+	capiMachineDeploymentGVR = clusterv1.GroupVersion.WithResource("machinedeployments")
+)
+
+// Infrastructure template GVRs are defined in capi_platform_helpers.go alongside their provider imports.
+
 const (
 	// Name of machine api namespace
```

golangci-lint (2.12.2)

[warning] exported: comment on exported const MachineAPINamespace should be of the form "MachineAPINamespace ..."

(revive)